### ISSUE TITLE
Version 0-9-1: false positive for SDTMIG-3.4 rule CORE-000594 for label "Reason Arm and/or Actual Arm is Null" for DM.ARMNRS

### ISSUE THREAD

**Issue by JozefAerts** [cdisc-org/cdisc-rules-engine#996]
With customer data (they are using SDTMIG-3-4) we got a false positive for label "Reason Arm and/or Actual Arm is Null" for DM.ARMNRS" stating "Variable label is not in title case.".
Might this be due to the slash (/) in the label?

**Comment by SFJohnson24** [comment]
i looked into the cache and got:
```
{'_links': {'modelDatasetVariable': {'href': '/mdr/sdtm/1-7/datasets/DM/variables/ARMNRS', 'title': 'Reason Arm and/or Actual Arm is Null', 'type': 'SDTM Dataset Variable'}, 'parentDataset': {'href': '/mdr/sdtmig/3-3/datasets/DM', 'title': 'Demographics', 'type': 'SDTM Dataset'}, 'parentProduct': {'href': '/mdr/sdtmig/3-3', 'title': 'Study Data Tabulation Model Implementation Guide: Human Clinical Trials Version 3.3 (Final)', 'type': 'Implementation Guide'}, 'rootItem': {'href': '/mdr/root/sdtmig/datasets/DM/variables/ARMNRS', 'title': 'Version-agnostic anchor resource for SDTMIG variable DM.ARMNRS', 'type': 'Root Data Element'}, 'self': {'href': '/mdr/sdtmig/3-3/datasets/DM/variables/ARMNRS', 'title': 'Reason Arm and/or Actual Arm is Null', 'type': 'SDTM Dataset Variable'}}, 'core': 'Exp', 'description': 'A coded reason that Arm variables (ARM and ARMCD) and/or actual Arm variables (ACTARM and ACTARMCD) are null. Examples: "SCREEN FAILURE", "NOT ASSIGNED", "ASSIGNED, NOT TREATED", "UNPLANNED TREATMENT". It is assumed that if the Arm and actual Arm variables are null, the same reason applies to both Arm and actual Arm.', 'label': 'Reason Arm and/or Actual Arm is Null', 'name': 'ARMNRS', 'ordinal': '26', 'role': 'Record Qualifier', 'simpleDatatype': 'Char'},
```
the label looks similar so I can only assume the / is confusing the validator for comparison.

**Comment by SFJohnson24** [comment]
this has been resolved with the changes to the regex in the rule--see JIRA ticket
